x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -506,24 +506,10 @@
     "common.ui.savedObjects.confirmModal.overwriteTitle": "{name} を上書きしますか？",
     "common.ui.savedObjects.confirmModal.saveDuplicateButtonLabel": "{name} を保存",
     "common.ui.savedObjects.confirmModal.saveDuplicateConfirmationMessage": "「{title}」というタイトルの {name} が既に存在します。保存を続けますか？",
-    "kibana-react.savedObjects.finder.filterButtonLabel": "タイプ",
-    "kibana-react.savedObjects.finder.searchPlaceholder": "検索…",
-    "kibana-react.savedObjects.finder.sortAsc": "昇順",
-    "kibana-react.savedObjects.finder.sortAuto": "ベストマッチ",
-    "kibana-react.savedObjects.finder.sortButtonLabel": "並べ替え",
-    "kibana-react.savedObjects.finder.sortDesc": "降順",
     "common.ui.savedObjects.howToSaveAsNewDescription": "Kibana の以前のバージョンでは、{savedObjectName} の名前を変更すると新しい名前でコピーが作成されました。今後この操作を行うには、「新規 {savedObjectName} として保存」を使用します。",
     "common.ui.savedObjects.overwriteRejectedDescription": "上書き確認が拒否されました",
     "common.ui.savedObjects.saveAsNewLabel": "新規 {savedObjectName} として保存",
     "common.ui.savedObjects.saveDuplicateRejectedDescription": "重複ファイルの保存確認が拒否されました",
-    "kibana-react.savedObjects.saveModal.cancelButtonLabel": "キャンセル",
```

Comment:
__note:__ this has been automatically rearranged by i18n script.

Reply:
Would it be worthwhile to find/replace `xpack.main.featureRegistry.*` with `xpack.features.*` in the translated files, since we know that the translated text doesn't need to change? Or is that not advised?

Reply:
We never did this since translators are supposed to receive en.json for every release, and in this case these labels will be recognized as already translated even though IDs changed (so that we're not charged additionally). But let's double check with @Bamieh

Reply:
Currently we send a json diff file in english and one for every language we have, so renaming these ID's will be synced with translators. We started doing this to sync translations introduced in PRs as we used to ignore them.

I'd vote for renaming these ids as it will also help keep kibana translated even before we sync with the translators which only happens a few days before every FF release.

Reply:
Okay, will do that than. I usually advised against manual edits in files with translations and treated them almost as "auto-generated" as it's easy to make something wrong there :)

Reply:
Done.